container/raw/handler.go

```diff
@@ -221,7 +227,12 @@ func (self *rawContainerHandler) getFsStats(stats *info.ContainerStats) error {
 }
 
 func (self *rawContainerHandler) GetStats() (*info.ContainerStats, error) {
-	stats, err := libcontainer.GetStats(self.cgroupManager, self.rootFs, os.Getpid(), self.ignoreMetrics)
+	pid := 0
+	// pid is used to collect network stats. Set it only for root container.
+	if self.isRootCgroup() {
+		pid = 1
```

Comment:
I see. I guess my comment about adding network metrics to the ignoreMetrics is optional in that case.